R/inputs-spectrum.R

```diff
@@ -89,17 +89,10 @@ extract_pjnz_one <- function(pjnz) {
 
   spec <- add_shiny90_unaware(spec, pjnz)
 
-
-  spectrum_region_code <- read_spectrum_region_code(pjnz)
-
-  if (spectrum_region_code == 0) {
-    spectrum_region_name <- eppasm::read_country(pjnz)
-  } else {
-    spectrum_region_name <- read_spectrum_region_name(pjnz)
-  }
```

Comment:
The hintr test data https://github.com/mrc-ide/hintr/blob/master/tests/testthat/testdata/Malawi2019.PJNZ relied on this conditional to work

```
> read_spectrum_region_name("tests/testthat/testdata/Malawi2019.PJNZ")
[1] NA
> eppasm::read_country("tests/testthat/testdata/Malawi2019.PJNZ")
[1] "Malawi"
```

The NA is causing problems further downstream when trying to generate output. Is this an issue of old test data in hintr or can we still get countries where the spectrum region code is 0?